WordPress/Jetpack/Info.plist

```diff
@@ -107,83 +19,6 @@
 			<key>UIPrerenderedIcon</key>
 			<true/>
 		</dict>
-		<key>CFBundleAlternateIcons</key>
-		<dict>
-			<key>WordPress Dark</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>wordpress_dark_icon_20pt</string>
-					<string>wordpress_dark_icon_29pt</string>
-					<string>wordpress_dark_icon_40pt</string>
-					<string>wordpress_dark_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-			</dict>
-			<key>Open Source</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>open_source_icon_20pt</string>
-					<string>open_source_icon_29pt</string>
-					<string>open_source_icon_40pt</string>
-					<string>open_source_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-				<key>WPRequiresBorder</key>
-				<true/>
-			</dict>
-			<key>Open Source Dark</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>open_source_dark_icon_20pt</string>
-					<string>open_source_dark_icon_29pt</string>
-					<string>open_source_dark_icon_40pt</string>
-					<string>open_source_dark_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-			</dict>
-			<key>Hot Pink</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>hot_pink_icon_20pt</string>
-					<string>hot_pink_icon_29pt</string>
-					<string>hot_pink_icon_40pt</string>
-					<string>hot_pink_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-			</dict>
-			<key>Jetpack Green</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>jetpack_green_icon_20pt</string>
-					<string>jetpack_green_icon_29pt</string>
-					<string>jetpack_green_icon_40pt</string>
-					<string>jetpack_green_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-			</dict>
-			<key>Pride</key>
-			<dict>
-				<key>CFBundleIconFiles</key>
-				<array>
-					<string>pride_icon_20pt</string>
-					<string>pride_icon_29pt</string>
-					<string>pride_icon_40pt</string>
-					<string>pride_icon_60pt</string>
-				</array>
-				<key>UIPrerenderedIcon</key>
-				<true/>
-			</dict>
-		</dict>
```

Comment:
I recently learned that Tuist supports code generation of the `Info.plist` file, too. [Here's a little experiment I made with it](https://github.com/mokagio/better-swift-testing/blob/trunk/Project.swift#L7-L36).

Dropping this here just for knowledge share and because `Info.plist` diffs are hard to read.

I believe having a tool generating them for us, with support for reusable blocks like Tuist does, would be useful. One day...